website/docs/foundations/colors/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,99 @@
+## Color styles and tokens
+
+Color naming and use can vary depending on context and tooling. This overview will clarify what naming conventions are typically used and how they align to the HDS standards implemented in our libraries.
+
+In Figma, reusable colors are referred to as “Styles” and are categorized and stored in the “style panel” to be referenced in fills, borders, and other properties that accept color values. Styles directly reference a HEX value but are unable to reference each other in Figma (known as aliasing in token terminology). This means when we provide Figma styles for semantic usage (like `Foreground/Primary` with a value of `#3B3D45`) a HEX value is referenced directly  instead of a base palette color (like `Palette/Neutral/Neutral 0` which is also `#3B3D45`). Even though these two are the same, Figma’s current configuration requires manual maintenance if any of the base values are updated.
```

Comment:
```suggestion
In Figma, reusable colors are referred to as “Styles” and are categorized and stored in the “style panel” to be referenced in fills, borders, and other properties that accept color values. Styles directly reference a HEX value but are unable to reference each other in Figma (known as aliasing in token terminology). This means when we provide Figma styles for semantic usage (like `Foreground / Primary` with a value of `#3B3D45`), the HEX value is referenced directly instead of a base palette color (like `Palette / Neutral / Neutral 0`, which is also `#3B3D45`). Even though these two are the same, Figma’s current configuration requires manual maintenance if any of the base values are updated.
```
For the other Foreground / ... instances, you have a space between the slash and the text. I noticed that this one doesn't have a space. Is this intentional, or should it follow the same pattern as the space?

Reply:
good catch, will be committing 👀